<launch>
    <arg name="ip" default="127.0.0.1" /> <!-- IP of the VRPN server (like Motive or Arena) -->
    <arg name="port" default="3883" /> <!-- Port of the VRPN server (default 3883) -->
    <arg name="name" default="human" />
    <arg name="use_timer" default="True" />
    <param name="kinect/person_name" type="string" value="$(arg name)" />
    
    <include file="$(find kinect_skeleton_publisher)/launch/kinect_skeleton_publisher.launch" >
      <arg name="ip" value="$(arg ip)"/> <!-- IP of the VRPN server (like Motive or Arena) -->
      <arg name="port" value="$(arg port)"/> <!-- Port of the VRPN server (default 3883) -->
      <arg name="num_skeletons" value="1"/>
    </include>

    <node name="kinect_calibration" pkg="kinect_skeleton_publisher" type="calibrate.py" output="screen" args="$(arg use_timer)" required="true"/>


</launch>
